app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java

```diff
@@ -505,11 +506,14 @@ private void setAttachmentClickListener(View view, StatusActionListener listener
     }
 
     private static CharSequence getAttachmentDescription(Context context, Attachment attachment) {
+        String duration = "";
+        if(attachment.getMeta().getDuration() != null && attachment.getMeta().getDuration() > 0) {
+            duration = formatDuration(attachment.getMeta().getDuration());
+        }
         if (TextUtils.isEmpty(attachment.getDescription())) {
-            return context
-                    .getString(R.string.description_status_media_no_description_placeholder);
+            return duration + " " + context.getString(R.string.description_status_media_no_description_placeholder);
         } else {
-            return attachment.getDescription();
+            return duration + " " + attachment.getDescription();
```

Comment:
We'll get an extra leading space here if the duration is empty, not sure how much it matters

Reply:
good catch, I will add a `trim()`